.gitignore

```diff
@@ -23,6 +23,6 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
-dist
```

Comment:
The compiled code shouldn't be committed to Github. I compile when creating a new release and push that to NPM. I think compiled code in Github is a bit icky as it creates weird diffs plus is hard to remember to keep updated plus isn't that useful anyways since you only need the compiled version when installing.